stock_vertical_lift_empty_tray_check/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_vertical_lift_operation_pick_zero_check_stock_user,access_vertical_lift_operation_pick_zero_check stock user,model_vertical_lift_operation_pick_zero_check,stock.group_stock_user,1,1,1,1
```

Comment:
Runbot is complaining about this, you should remove the file:

    2021-03-11 17:53:03,262 183 WARNING openerp_test odoo.modules.loading: The transient model vertical.lift.operation.pick.zero.check (Make sure the tray location is empty) should not have explicit access rules!